packages/commonwealth/client/scripts/views/pages/Communities/AllTabContent.tsx

```diff
@@ -54,124 +60,203 @@ const AllTabContent: React.FC<AllTabContentProps> = ({
+          </div>
+        </div>
+        <TrendingThreadList
+          query={useFetchGlobalActivityQuery}
+          customScrollParent={containerRef.current}
+          hideHeader
+        />
       </div>
 
       {/* Users section */}
       <div className="section-container">
-        <CWText type="h2">Users</CWText>
+        <div className="heading-container">
+          <CWText type="h2">Users</CWText>
+          <div
+            className="link-right"
+            onClick={() => navigate('/explore?tab=users')}
+          >
+            <CWText className="link">See all users</CWText>
+            <CWIcon iconName="arrowRightPhosphor" className="blue-icon" />
+          </div>
+        </div>
         <div className="users-xp-table">
-          <XPEarningsTable />
+          <XPTable />
         </div>
       </div>
-      {launchpadEnabled && !hideHeader && (
-        <CWText type="h2">Communities</CWText>
-      )}
-      {isLoading && communitiesList.length === 0 ? (
-        <CWCircleMultiplySpinner />
-      ) : (
-        <Virtuoso
-          key={Object.values(filters)
-            .map((v) => `${v}`)
-            .join('-')}
-          className="communities-list"
-          style={{ height: '100%', width: '100%' }}
-          data={isInitialCommunitiesLoading ? [] : communitiesList}
-          customScrollParent={containerRef.current}
-          itemContent={(listIndex, slicedCommunities) => {
-            return slicedCommunities.map((community, sliceIndex) => {
-              const canBuyStake = !!user.addresses.find?.(
-                (address) => address?.community?.base === community?.base,
-              );
 
-              const historicalPriceMap: Map<string, string | undefined> =
-                new Map(
-                  Object.entries(
-                    (historicalPrices || [])?.reduce(
-                      (
-                        acc: Record<string, string | undefined>,
-                        { community_id, old_price },
-                      ) => {
-                        acc[community_id] = old_price || undefined;
-                        return acc;
-                      },
-                      {},
-                    ),
-                  ),
+      {/* Communities section */}
+      <div className="section-container">
+        {launchpadEnabled && !hideHeader && (
+          <div className="heading-container">
+            <CWText type="h2">Communities</CWText>
+            <div
+              className="link-right"
+              onClick={() => navigate('/explore?tab=communities')}
+            >
+              <CWText className="link">See all communities</CWText>
+              <CWIcon iconName="arrowRightPhosphor" className="blue-icon" />
+            </div>
+          </div>
+        )}
+
+        {isLoading && communitiesList.length === 0 ? (
+          <CWCircleMultiplySpinner />
+        ) : (
+          <Virtuoso
```

Comment:
also this and CommunitiesTabContent is exactly the same code, it would be nice to reuse it

Reply:
hey @masvelio I actually refactor the UI to use a different component in #11434 so I won't resolve this one here
- Basically in that PR in AllTabsContent.tsx I use `TrendingCommunitiesPreview` here instead so there's no copied code.